src/convertToObject.js

```diff
@@ -12,7 +12,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssProperties = sourceString.split(';');
+
+  return cssProperties.reduce((object, property) => {
+    const [key, value] = property.split(':');
+
+    if (!key.length || !value) {
+      return object;
+    }
+
+    object[key.trim()] = value.trim();
+
+    return object;
+  }, {});
```

Comment:
```suggestion
	 if (key && value) {
			object[key.trim()] = value.trim();
	 }
	  
	 return object;
  }, {});
```
	I would check key and value existense instead of their absence.

Reply:
Непоганий комент, погоджуюся з Андрієм